Woopra/Woopra-iOS
# Support typed value in the body of POST request

### Purpose:
1. Support typed value instead of converting the value to String.
2. A small refactor has been made to align the code between `WTracker` and `WIdentify`

### Expected Behavior:
Both `WTracker` and `WIdentify` are still functioning as they did before.

### How To Verify it:
Make a push call.
```swift
WTracker.shared.visitor.add(property: "name", value: "new-user-name")
WTracker.shared.visitor.add(property: "email", value: "new-email@mail.com")
WTracker.shared.push()
```

Start a track event and check if it appears on the dashboard.
```swift
let event = WEvent.event(name: "event-name")
event.add(property: "view", value: "view-value")
WTracker.shared.trackEvent(event)
```
